src/modules/submissions.js

```diff
@@ -499,6 +499,8 @@
 			];
 
 		if ( isAccept ) {
+			// Remove Draft article templates
+			text = text.replace( /\{\{Draft(?: article(?:\|(?:subject=)?foo)?|\|(?:subject=)?foo)?\}\}/gi, '' );
```

Comment:
Please have it use any word (`[^\|]+`) instead of "foo", and then it should match the new case I just added. ([The template documentation](https://en.wikipedia.org/wiki/Template:Draft_article) says people can put any article title for the first parameter (the text after the `|` if there's no `=`).

Also, it looks like it removed two of the templates under "none of these should be removed" at [Draft:Testing AFCH](https://test.wikipedia.org/wiki/Draft:Testing_AFCH). If the `text` parameter is used, the template shouldn't be removed, since people could put any text in there, including wikitext and other templates (so we couldn't just stop searching at the first }}). I'd recommend either trying [negative lookahead](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Assertions#other_assertions) or [using a function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_function_as_a_parameter) for the .replace function to test for the substring `|text=` (regex `\|\s*text\s*=`, probably).